.github/workflows/test.yml

```diff
@@ -1,35 +1,35 @@
 name: CI - Test
 on:
-  push:
-    branches:
-      - '*'
-  pull_request:
-    branches:
-      - '*'
+push:
+branches:
+- '*'
+pull_request:
+branches:
+- '*'
```

Comment:
In general for this file: are the changes intentional? It looks like there are structural changes to the yaml file here.

Reply:
This is an issue with prettier when I run format, I'm not sure I'll compare the changes before this was modified and revert back.